src/app/components/RoundedBalance/index.tsx

```diff
@@ -24,6 +25,7 @@ export const RoundedBalance: FC<RoundedBalanceProps> = ({
   scope,
   tokenAddress,
   tickerAsLink,
+  roundLargeNumbers,
```

Comment:
Needs a different name (this isnt rounding) (although the whole RoundedBalance isnt rounding either, it's truncating)